index.html

```diff
@@ -0,0 +1,379 @@
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <!-- Contact form for Code Review Exercise -->
+      <section class="contact-div ">
+        <p class="contact-text list-title">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/mrgjojra" method="post" class="form flex">
+          <input type="text" placeholder="Full name" name="full name" id="full_name" max="30" required class="input no-border">
+          <hr class="liner">
+          <input type="text" name="email" placeholder="Email address" id="email" required class="input no-border">
+          <hr class="liner">
+          <textarea name="comment" id="texter" maxlength="500" class="no-border" reqiured>Enter text here</textarea>
+          <button class="contact_button no-border link"Get in touch</button>
```

Comment:
You did well 👌💯 at adding the button, but I think you forgot to add the angle bracket ">" to close the start🎈 tag button, but you did well 😊, so you can fix this to enhance your code I will show you below ⤵️ your button 🪄and the button at Figma.

Your design: ⤵️


![Screenshot_20230106_124246](https://user-images.githubusercontent.com/83360636/210894543-51e0f77e-44f1-425f-a4f9-4108e3ab49ee.png)

The Figma design: ⤵️

![Screenshot_20230106_124220](https://user-images.githubusercontent.com/83360636/210894710-a19f8571-3221-471a-bc0b-5e931ebd11ee.png)